### PR TITLE
Fixes grammar and spelling everywhere

### PR DESCRIPTION
**Pull request by Teleported-Bread**
## About The Pull Request
Fixes grammar and spelling across 200 files. 

## Why It's Good For The Game
Bad spelling bad, good spelling good.

## Changelog

:cl:
spellcheck:Fixes spelling and grammar all over the place. It's is the contraction, and receiving is the spelling!
/:cl:

### CODE REVIEW

**Comment by Teleported-Bread** [comment]
okay what in gods name happened here, i messed something up bad

**Comment by TheVekter** [comment]
This is a good second PR, but you've got some issues that need to be resolved.

1) I haven't gone through every last change to make sure it's the case, but did you go through and just do a "replace its with it's" on this, or did you manually check each one to be sure it was grammatically correct?
2) You've got conflicts, which is kind of inevitable when you're touching this many files. Someone who is better versed in Git can probably help you better than I can on how to resolve these.

**Comment by Teleported-Bread** [comment]
I manually checked the its/it's, and it was not pleasant but i got it done.

File: code/_onclick/hud/rendering/_render_readme.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ When you create an atom, this will always create an internal byond structure cal
 
 Finally, we arrive at clientside behavior, where we have two main clientside functions: GetMapIcons, and Render. GetMapIcons is repsonsible for actual rendering calculations on the clientside, such as "Group Icons and Set bounds", which performs clientside calculations for transform matrixes. Note that particles here are handled in a separate thread and are not diplayed in the clientside profiler. Render handles the actual drawing of the screen.
 
-For debugging rendering issues its reccomended you do two things:
+For debugging rendering issues it's reccomended you do two things:
```
Comment (Fikou) [inline code/_onclick/hud/rendering/_render_readme.md:15]:
```suggestion
For debugging rendering issues it's recommended you do two things:
```
sheesh

File: code/modules/mapfluff/ruins/spaceruin_code/TheDerelict.dm [hunk 2]
```diff
@@ -191,7 +191,7 @@
 		"\[00:01\] Every single day I spend on this station",
 		"\[00:07\] I become more and more certain that this was some kind of macabre punishment assignment",
 		"\[00:17\] I don't know what I did to deserve being consigned to this glued together ball of orbital wreckage",
-		"\[00:28\] But someones got it out for me",
+		"\[00:28\] But someone has it out for me",
```
Comment (dragomagol) [inline code/modules/mapfluff/ruins/spaceruin_code/TheDerelict.dm:194]: This one isn't an error, just a variant. https://idioms.thefreedictionary.com/have+got+it+in+for+(one)
Should be `someone's got it out for me`

File: code/modules/admin/view_variables/reference_tracking.dm [hunk 3]
```diff
@@ -43,7 +43,7 @@
 	DoSearchVar(global_vars, "Native Global", search_time = starting_time)
 	log_reftracker("Finished searching native globals")
 
-	for(var/datum/thing in world) //atoms (don't beleive its lies)
+	for(var/datum/thing in world) //atoms (don't belve its lies)
```
Comment (Sealed101) [inline code/modules/admin/view_variables/reference_tracking.dm:46]:
```suggestion
	for(var/datum/thing in world) //atoms (don't believe its lies)
```
she believed
he lied
sbeve

File: code/modules/surgery/organs/external/restyling.dm [hunk 4]
```diff
@@ -41,7 +41,7 @@
 			target_organ = valid_features[1]
 		if(2 to INFINITY)
 			var/choose_options = list()
-			var/name_to_organ = list() //literally so I dont have to loop again after someones made their choice
+			var/name_to_organ = list() //literally so I dont have to loop again after someone made their choice
```
Comment (dragomagol) [inline code/modules/surgery/organs/external/restyling.dm:44]:
```suggestion
			var/name_to_organ = list() //literally so I dont have to loop again after someone's made their choice
```

File: code/modules/antagonists/traitor/objectives/final_objective/romerol.dm [hunk 5]
```diff
@@ -1,11 +1,11 @@
 /datum/traitor_objective/ultimate/romerol
 	name = "Spread the experimental bioterror agent Romerol by calling a droppod down at %AREA%"
-	description = "Go to %AREA%, and recieve the bioterror agent. Spread it to the crew, \
+	description = "Go to %AREA%, and receive the bioterror agent. Spread it to the crew, \
 	and watch then raise from the dead as mindless killing machines. Warning: The undead will attack you too."
```
Comment (Sealed101) [inline code/modules/antagonists/traitor/objectives/final_objective/romerol.dm:4]:
```suggestion
	description = "Go to %AREA%, and receive the bioterror agent. Spread it to the crew, \
	and watch them rise from the dead as mindless killing machines. Warning: The undead will attack you too."
```
while we're here

File: html/changelogs/archive/2011-07.yml [hunk 6]
```diff
@@ -72,7 +72,7 @@
       have been removed. This doesn't mean admins can't play midis, this just gets
       rid of a lot of unnecessary download time.
   Rastaf.Zero:
-  - unknown: 'Botanists get a new toy: Biogenerator. Insert biological items, recieve
+  - unknown: 'Botanists get a new toy: Biogenerator. Insert biological items, receive
```
Comment (dragomagol) [inline html/changelogs/archive/2011-07.yml:75]: Don't change old changelog entries

File: strings/tips.txt [hunk 7]
```diff
@@ -129,7 +129,7 @@ As an Atmospheric Technician, your ATMOS holofan projector blocks gases while al
 As an Atmospheric Technician, your backpack firefighter tank can launch resin. This resin will extinguish fires and replace any gases with a safe, room-temperature airmix.
 As an Engineer, return to Engineering once in a while to check on the engine and SMES cells. It's always a good idea to make sure the Supermatter isn't delaminating.
 As an Engineer, the Supermatter Monitoring Program on modular computers give you a detailed report on the Supermatter's condition, and the contents of the air inside of the chamber, allowing you to both diagnose and monitor programs from afar!
-As an Engineer, the Supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you. So will touching it with telepathy.
+As an Engineer, the Supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you. So will touching it with telekinesis.
```
Comment (dragomagol) [inline strings/tips.txt:132]: Didn't we just merge a PR that does this?

File: code/game/objects/items/devices/radio/intercom.dm [hunk 8]
```diff
@@ -10,7 +10,7 @@
 	item_flags = NO_BLOOD_ON_ITEM
 
 	overlay_speaker_idle = "intercom_s"
-	overlay_speaker_active = "intercom_recieve"
+	overlay_speaker_active = "intercom_receive"
```
Comment (dragomagol) [inline code/game/objects/items/devices/radio/intercom.dm:13]: Did you make sure this won't cause any issues?

**Comment by Sealed101** [comment]
i was mid-review goddamnit

**Comment by dragomagol** [comment]
ME TOO

**Review by dragomagol — COMMENTED** [review]
I'm sending this anyway because I already wrote it

**Review by Sealed101 — CHANGES_REQUESTED** [review]
### i too was mid-review

if you are modifying `icon_state`s make sure to go to the `icon`'s .dmi file and change the name of the icon state there as well

changelogs don't need changing doe

it's kinda funny how most of these are just code comments but w/e

**Comment by ZephyrTFA** [comment]
Really don't appreciate how you came in, mass modified hundreds of files, and then closed the PR the second you got any feedback.

**Comment by Teleported-Bread** [comment]
I made an awful pr, I shouldn't have made it, didn't even plan to upload it in the form it was in, and when I realized how severely broken it was I closed it. If i get back to this, i'll be doing it in a completely new PR that uh... isn't completely awful.

**Pull request closed without merging**